### PR TITLE
Feat/help docs

### PR DESCRIPTION
**Pull request by AlecAivazis**
This PR adds documentation for the recent features added by @coryb. Once this get in, i'm going to cut a release so `gopkg.in` will pick it up. Thanks again for all the hard work!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +163,58 @@ validators include:
...
+
+Customizing the icons and various parts of survey can easily be done by setting the following variables
+in `survey/core`:
+
+|   name                 |     default    |    description                                                    |
+|------------------------|----------------|-------------------------------------------------------------------|
+| ErrorIcon              |       ✘        | Before an error                                                   |
+| HelpIcon               |       ⓘ       | Before help text                                                   |
+| QuestionIcon           |       ?        | Before the message of a prompt                                    |
+| MultiSelectFocusIcon   |       ❯        | Marks the current selection in `Select` and `MultiSelect` prompts |
```
Comment (coryb) [inline README.md]: I renamed it to `SelectFocusIcon` for both.

File: README.md [hunk 2]
```diff
@@ -156,6 +163,58 @@ validators include:
 | MinLength(n) |   string        |   Enforces that a response is at least the given length       |
 | MaxLength(n) |   string        |   Enforces that a response is no longer than the given length |
 
+## Help Text
+
+All of the prompts have a `Help` field which can be defined to provide more information to your users:
```
Comment (coryb) [inline README.md]: Minor nitpick, `Password` does not have `Help`, but not sure it is worth calling out.

Comment (AlecAivazis) [inline README.md]: Probably worth mentioning

**Comment by AlecAivazis** [comment]
sweet - cutting release now

**Pull request merged**